drivers/gae/src/main/java/org/jclouds/gae/GaeHttpCommandExecutorService.java

```
@@ -88,4 +89,9 @@ protected void cleanup(HTTPRequest nativeRequest) {
    protected HttpResponse invoke(HTTPRequest request) throws IOException {
       return convert(urlFetchService.fetch(request));
    }
+
+   @Override
+   protected ListenableFuture<HttpResponse> invokeAsync(final HTTPRequest nativeRequest) {
+      return null;
```

Lets throw unsupported exceptions here too